packages/next-auth/src/core/routes/signin.ts

```diff
@@ -59,7 +59,6 @@ export default async function signin(params: {
 
     // Check if user is allowed to sign in
     try {
-      // @ts-expect-error
```

Comment:
Could we add a `TODO:` comment explaining why this was skipped?

Reply:
@lluia I don't get this comment, could you help clarify it a bit? 🙌